http/org.wso2.carbon.transport.http.netty/src/main/java/org/wso2/carbon/transport/http/netty/sender/websocket/WebSocketClient.java

```diff
@@ -0,0 +1,252 @@
+/*
```

Comment:
It is better if we add java doc comments describing the methods for the public methods. WDYT?